fms/models/hf/utils.py

```diff
@@ -179,7 +179,7 @@ def as_fms_model(
         architecture = "llama"
```

Comment:
leaving this comment here, but the best way to load the GPTQ models, given they are already in HF format coming from AutoGPTQ might be using this function and loading the GPTQ info from the json here